_opcodes/op066-iwd1.html

```diff
@@ -19,4 +19,9 @@
 <br/>0 Default
 <br/>1 Fade In
 <br/>2 Fade Out
-<br/>This effect sets stat #60 (TRANSLUCENT) to param #1.
+
+{% capture note %}
+This effect sets stat #60 (TRANSLUCENT) to param #1.
+{% endcapture %}
```

Comment:
these should just remain as is. There's nothing special or extra about the stat being set to warrant more attention from the viewer.